### PR TITLE
[NEW] procurement_analytic: New module

### PR DESCRIPTION
**Pull request by carlosdauden**
This PR replaces https://github.com/OCA/account-analytic/pull/61

### CODE REVIEW

File: procurement_analytic/README.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/87/8.0
+
+Known issues / Roadmap
+======================
+
+* If product supplier info min quantity is greater than procurement qty and we
+  have sale orders with distinct analytic account witch contains this product,
```
Comment (pedrobaeza) [inline procurement_analytic/README.rst]: s/witch/which

File: procurement_analytic/README.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+#. Set *Analytic Account* in *Extra Information* tab.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/87/8.0
+
+Known issues / Roadmap
+======================
+
+* If product supplier info min quantity is greater than procurement qty and we
```
Comment (pedrobaeza) [inline procurement_analytic/README.rst]: This issue is not for this module, but for the purchase one.

File: procurement_analytic/README.rst [hunk 3]
```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Procurement Analytic
+====================
+
+This module adds account analytic field in procurements.
```
Comment (pedrobaeza) [inline procurement_analytic/README.rst]: Add:

```
This field can be used for propagating this data to the generated objects, like purchase orders or manufacturing orders.
```

**Comment by carlosdauden** [comment]
Changes done

**Comment by pedrobaeza** [comment]
Why there isn't Travis?

Anyways, :+1:

**Comment by andhit-r** [comment]
Code review :+1:

File: procurement_analytic/views/procurement_analytic.xml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+<openerp>
+<data>
+
+<record id="procurement_form_view_analiytic" model="ir.ui.view">
+    <field name="name">procurement_purchase_analytic.procurement.order.form</field>
+    <field name="model">procurement.order</field>
+    <field name="inherit_id" ref="procurement.procurement_form_view"/>
+    <field name="arch" type="xml">
+        <field name="group_id" position="after">
+            <field name="account_analytic_id" groups="purchase.group_analytic_accounting"/>
```
Comment (yajo) [inline procurement_analytic/views/procurement_analytic.xml:13]: Sorry to be late here. `purchase` is not in the dependency graph AFAICS, shouldn't this be `analytic.group_analytic_accounting`?

Comment (pedrobaeza) [reply procurement_analytic/views/procurement_analytic.xml:13]: Yes, this has been already discussed with @carlosdauden, and will be changed next week.

**Pull request merged**